BepInEx.MessageCenter/Properties/AssemblyInfo.cs

```diff
@@ -1,35 +1,15 @@
-﻿using System.Reflection;
-using System.Runtime.InteropServices;
-
-// General Information about an assembly is controlled through the following
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
-[assembly: AssemblyTitle("BepInEx.MessageCenter")]
-[assembly: AssemblyDescription("")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("https://github.com/BepInEx/BepInEx.Utility")]
-[assembly: AssemblyProduct("BepInEx.MessageCenter")]
-[assembly: AssemblyCopyright("Copyright ©  2019")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyCulture("")]
-
-// Setting ComVisible to false makes the types in this assembly not visible
-// to COM components.  If you need to access a type in this assembly from
-// COM, set the ComVisible attribute to true on that type.
-[assembly: ComVisible(false)]
-
-// The following GUID is for the ID of the typelib if this project is exposed to COM
-[assembly: Guid("49f9a640-b592-4a4f-9b73-c8c6b8f28c22")]
-
-// Version information for an assembly consists of the following four values:
-//
-//      Major Version
-//      Minor Version
-//      Build Number
-//      Revision
-//
-// You can specify all the values or you can default the Build and Revision Numbers
-// by using the '*' as shown below:
-// [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion(BepInEx.MessageCenter.Version)]
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+using static BepInEx.MessageCenter;
+
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("https://github.com/BepInEx/BepInEx.Utility")]
+[assembly: AssemblyCopyright("Copyright 2020")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+[assembly: ComVisible(false)]
+[assembly: AssemblyTitle(GUID)]
+[assembly: AssemblyProduct(GUID)]
+[assembly: AssemblyDescription(PluginName)]
+[assembly: AssemblyVersion(Version)]
+[assembly: AssemblyFileVersion(Version)]
```

Comment:
In this project each plugin has an unique version number, and each plugin class that contains the constant is named differently so a shared assinfo isn't really viable. Removing the excess is a good idea though.

Reply:
You can move the version info under the same class name under every single project, like `PluginInfo`. In that case, every project has that class with the same constants but different values. The shared project can then use that info fine.
It's a pretty small refactor which doesn't affect how the plugin itself works since constants are not part of some critical API.

Reply:
Fixed